tools/snp.sh

```diff
@@ -1235,12 +1322,19 @@ main() {
       ;;
 
     attest-guest)
+	if [ "$SVSM" = true ]; then
       install_rust
       install_sev_snp_measure
       install_dependencies
       wait_and_retry_command verify_snp_guest
-      setup_guest_attestation
-      attest_guest
+	  setup_guest_attestation
+	  attest_guest
+	else
+	  wait_and_retry_command verify_snp_guest
+	  setup_svsm_guest_attestation
+	  attest_guest
+    fi
+     attest_guest  	  
```

Comment:
Indentation here seems really off. Are you using tabs vs spaces? Also, if we are repeating all but one line, we should adjust this conditional logic.